templates_ark/example_pixel_clustering.ipynb

```diff
@@ -767,11 +767,38 @@
     "with open(os.path.join(base_dir, pixel_output_dir, '%s_cell_clustering_params.json' % pixel_cluster_prefix), 'w') as fh:\n",
```

Comment:
<p>We know where the TIFs are already from processing them in this notebook, no need for a fake path.</p><p>The mask suffix should be pixel, not cell, since this is the pixel notebook. Same thing for mapping path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>